containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Setting global arguments
+ARG BUNDLE_WITH=sqlite:mysql:postgres
+ARG BUNDLE_WITHOUT=development:test:private
+ARG BUNDLE_DEPLOYMENT=true 
+
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+# include global args
+ARG BUNDLE_WITH
+ARG BUNDLE_WITHOUT
+ARG BUNDLE_DEPLOYMENT
+
+LABEL org.opencontainers.image.authors ='pglombardo@hey.com'
+
+# Required packages
+RUN apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
```

Comment:
> Fix hadolint error.

 Why is this an error at all? 
Docker docs say each `ENV` will create an additional intermediate layer.  Therefore it should be better to have a single `ENV` command then multiple.

Reply:
> Btw. Why is `PATH` and `HOME` exposed? Running as `pwpush` user has home already set to `APP_ROOT` I suspect? Just asking.

They aren't exposed. They're only set within docker image. 
But if it is necessary to set them at all should/may be answered by @pglombardo. This may be related to ruby or so.

Eitherway the next days I'll try and build the image without setting those and see what's gonna happen .

Reply:
https://github.com/hadolint/hadolint/wiki/DL3044

I think it is more specific to the fact that `APP_ROOT` is used on the same line as an input for an another variable later. 🤷🏻‍♂️

Reply:
> Eitherway the next days I'll try and build the image without setting those and see what's gonna happen .

I'm happy to help out if you need something. 👍🏻  Teamwork makes dreams work, and all that. 😇

Reply:
Removing both `HOME` and `PATH` did not change anything from my current build.  I tried both as ephemeral and MariaDB (compose).  Both started up and I was able to submit passords.  They too showed up in the database.

Reply:
This (within your refernce link)

> https://github.com/hadolint/hadolint/wiki/DL3044
> 
> I think it is more specific to the fact that `APP_ROOT` is used on the same line as an input for an another variable later. 🤷🏻‍♂️

confirms 

> Removing both `HOME` and `PATH` did not change anything from my current build. I tried both as ephemeral and MariaDB (compose). Both started up and I was able to submit passords. They too showed up in the database.

in my eyes.

I'll try to confirm this. If so I will remove both statements.

Reply:
I can confirm both `HOME` and `PATH` are not necessary for building nor for running the images.
As a consequence they were removed.